corehq/form_processor/parsers/form.py

```diff
@@ -214,16 +214,7 @@ def apply_deprecation(existing_xform, new_xform, interface=None):
     #  - Save the new instance to the previous document to preserve the ID
 
     interface = interface or FormProcessorInterface(existing_xform.domain)
-
-    if existing_xform.persistent_blobs:
-        for name, meta in existing_xform.persistent_blobs.items():
-            with existing_xform.fetch_attachment(name, stream=True) as content:
-                existing_xform.deferred_put_attachment(
```

Comment:
where these calls failing on SQL forms in the past? If not, why didn't it work for SQL forms?